tracee-ebpf/tracee/tracee.go

```diff
@@ -918,6 +923,24 @@ func (t *Tracee) writeProfilerStats(wr io.Writer) error {
 	return nil
 }
 
+func (t *Tracee) getProcessCtx(hostTid int) (ProcessCtx, error) {
+	processCtx, procExist := t.processTree.processTreeMap[hostTid]
+	if procExist {
+		return processCtx, nil
+	} else {
+		processContextMap, err := t.bpfModule.GetMap("process_context_map")
+		if err != nil {
+			return processCtx, err
+		}
+		processCtxBpfMap, err := processContextMap.GetValue(unsafe.Pointer(&hostTid))
+		if err != nil {
+			return processCtx, err
+		}
+		processCtx, err = t.ParseProcessContext(processCtxBpfMap)
```

Comment:
Won't it be better to update processTreeMap before returning, so we will not have another cache miss?

Reply:
yep i just added it